interfaces/matlab/toolbox/importPhase.m

```diff
@@ -13,6 +14,7 @@
 %     Instance of class :mat:func:`Solution`
 %
 
+warning('This function is deprecated and will be removed after Cantera 2.3.');
```

Comment:
@speth, @bryanwweber, before I make the change would this be an appropriate modification?
 
warning('This function is deprecated and will be removed after Cantera 2.3. ...
reference function s = Solution(src, id, trans)');